UnitsNet.Tests/CustomCode/ReciprocalLengthTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using UnitsNet.Units;
+using Xunit;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class ReciprocalLengthTests : ReciprocalLengthTestsBase
+    {
+        protected override bool SupportsSIUnitSystem => false;
+
+        protected override double InverseMeterInOneInverseMeter => 1;
+        protected override double InverseCentimeterInOneInverseMeter => 1E-2;
+        protected override double InverseMillimeterInOneInverseMeter => 1E-3;
+
+        protected override double InverseMileInOneInverseMeter => 1 / 0.000621371;
```

Comment:
Please use constants looked up in an external reference, instead of calculations.

This is meant to serve as a double check that we got it right, since it is very easy to get it inverse and still pass all tests.

According to this converter, InverseMileInOneInverseMeter should be `1609.344`.
https://www.unitsconverters.com/en/Reciprocal-Length-Conversions/Measurement-1182

Reply:
You're right. I was not sure at the beginning and there is an inconsistency throughout the repo so I've picked the fraction way. Now fixed.